share/default_regions.py

```diff
@@ -1,44 +1,44 @@
 import cdutil
 
 regions_specs = {
-    'NHEX': {'domain': cdutil.region.domain(latitude=(30., 90, 'ccb'))},
```

Comment:
@doutriaux1 why was this truncated?

Reply:
No idea. Let me put it back in. Nice catch.